src/app/ReadHandler.h

```diff
@@ -221,6 +226,8 @@ class ReadHandler : public Messaging::ExchangeDelegate
     AttributePathExpandIterator mAttributePathExpandIterator = AttributePathExpandIterator(nullptr);
     bool mIsFabricFiltered                                   = false;
     bool mHoldSync                                           = false;
+    // The detailed encoding state for a single attribute, used by list chunking feature.
+    AttributeValueEncoder::AttributeEncodeState mAttributeEncoderState = AttributeValueEncoder::AttributeEncodeState();
```

Comment:
Why are we doing the `= defaulconstructor` thing?  If we just take all that out, the member will get default-constructed....